Mapping/ClassMetadata.php

```diff
@@ -106,6 +115,22 @@ public function getIri()
     }
 
     /**
+     * @return string
+     */
+    public function getIdentifier()
+    {
+        return $this->identifier;
```

Comment:
Agree with @dunglas, these classes must stay away each other.

@MLKiiwy we don't need this new "own annotation" anymore... If someone want to use another backend than Doctrine, it'll write its own loader that'll identify the class identifier, that's all :)

If there's no `id` attribute, one of the `AttributeMetadata` must be "tagged" as an identifier. If it's not the case, then throw an exception as the class is not "identifiable".

Can you explain why this loader order ? Because when we start to have this kind of rules, I'm afraid about high complexity and low extensibility...